routes/clams.js

```diff
@@ -0,0 +1,136 @@
+const express = require('express');
+const { pool } = require('../server/db');
+const { keysToCamel } = require('../common/utils');
+
+const router = express.Router();
+
+// Get data for all clams
+router.get('/', async (req, res) => {
+  try {
+    const clams = await pool.query('SELECT * FROM clams;');
+    res.status(200).json(keysToCamel(clams));
+  } catch (err) {
+    res.status(400).send(err.message);
+  }
+});
+
+// Get data for specific clam based on clamId
+router.get('/:clamId', async (req, res) => {
+  try {
+    const clam = await pool.query('SELECT * FROM clams WHERE clamId = $(clamId);');
+    res.status(200).json(keysToCamel(clam.rows));
+  } catch (err) {
+    res.status(400).send(err.message);
+  }
+});
+
+// Get data for clams based on rakerId
+router.get('/rakers/:rakerId/clams', async (req, res) => {
```

Comment:
this route scheme is a little long, since we're getting a raker's clams from `/clams` already you may be able to shorten this to something like `/clams/raker/:rakerId`